tests/test_livysession.py

```diff
@@ -39,7 +39,8 @@ class TestLivySession:
     ready_statement_json = json.loads('{"id":0,"state":"available","output":{"status":"ok","execution_count":0,"data":{"text/plain":"Pi is roughly 3.14336"}}}')
     log_json = json.loads('{"id":6,"from":0,"total":212,"log":["hi","hi"]}')
 
-    def __init__(self):
+    def __init__(self, methodName="test_create_sql_hive_context_happens_once"):
```

Comment:
I'm inheriting unittest class and this is the way to call the super class by specifying the method that you apply your test case to.